packages/pigeon/example/README.md

```diff
@@ -115,27 +115,25 @@ Future<bool> sendMessage(String messageText) {
 ### Swift
 
 This is the code that will use the generated Swift code to receive calls from Flutter.
-packages/pigeon/example/app/ios/Runner/AppDelegate.swift
+Unlike other languages, when throwing an error (both synchronous and asynchronous), use `PigeonError` instead of `FlutterError`, as `FlutterError` is not conforming to `Swift.Error`.
+Previously, a workaround was to declare an extension to `FlutterError` to conform to `Swift.Error`, but, as of Pigeon 19.0.0, `PigeonError` is provided for this purpose. Older code still using `FlutterError` will continue to work, but it is recommended to switch to `PigeonError` and remove the extension.
```

Comment:
`Unlike other languages, when throwing an error, use `PigeonError` instead of `FlutterError`,
as `FlutterError` does not conform to `Swift.Error`.

I don't think the second line is necessary.

Reply:
Done.